### ISSUE TITLE
Don't run with php 8

### ISSUE THREAD

**Issue by tempral** [phpLiteAdmin/pla#28]
Some break changes?
Maybe is too early to say that

**Comment by popey456963** [comment]
Should be added to documentation, ran into `MicroTimer` issues.

**Comment by Dygear** [comment]
I've been working on a V2 version of phpLiteAdmin with PHP 8.0.0 as the target version. It's by no means feature complete, but it's an intresting toy right now. I am using it to admin some of my databases on a server that only runs PHP 8.x that's about to be deployed into production and it's been "good enough" for me to get by for the moment. It does some self mutation in order to add users or databases, so you the end user shouldn't need to go into the source code ever really.


https://github.com/staticflux/pla/tree/v2-dygear

**Comment by crazy4chrissi** [comment]
What exactly is not working in PHP 8? Current development version 1.9.9-dev seems to be working perfectly for me with PHP 8.0.3 on Windows.

You get the current development build here: https://www.phpliteadmin.org/phpliteadmin-dev.zip

**Comment by tempral** [comment]
thanks, I will try soon the dev version

**Comment by mircerlancerous** [comment]
`each` function is removed in php 8.0.0
https://www.php.net/manual/en/function.each.php

**Comment by crazy4chrissi** [comment]
Yes. The development version does not use `each` and works in PHP 8.
https://www.phpliteadmin.org/phpliteadmin-dev.zip
